blocks/library/embed/editor.scss

```diff
@@ -1,7 +1,20 @@
+.editor-block-list__block[data-type="core/embed"] {
+	&[data-align="left"],
+	&[data-align="right"] {
+		max-width: 100%;
```

Comment:
Is this necessary? Or at least, should it be applying so specifically to the embed block type? Why aren't other aligned blocks susceptible to this?

https://github.com/WordPress/gutenberg/blob/d5711dc5191fba29630a88a270ebc69c3823ea4b/editor/components/block-list/style.scss#L112-L118

In my testing, it seems to work fine without this specific style.

Reply:
Actually, from what I can tell there's an issue of [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) where the `max-width: 700px` of `.editor-visual-editor .editor-block-list__block` takes precedence over the `max-width: 370px` of `.editor-block-list__block[data-align="left"]`

https://github.com/WordPress/gutenberg/blob/d5711dc5191fba29630a88a270ebc69c3823ea4b/editor/edit-post/modes/visual-editor/style.scss#L17-L40

Might have been a regression accidentally introduced with changes of #3546.

Reply:
It's only necessary to make aligned embed responsive. It's visible only at < 320px, I'm not sure if it should be supported.

Regarding specificity issue, the `max-width` is not related to fixing aligned embed videos, so it's probably unnecessary.